app/views/dashboard/index.html.haml

```diff
@@ -0,0 +1,35 @@
+/ FIRST ROW OF BLOCKS
+.row
+  / CARS BLOCK
+  .col-sm-4.col-lg-3
```

Comment:
Suggestion: If you could create a partial for a 'Block' with two parameters: title, partial_name it would be great. The second parameter should be provide to a partial with the content of the given section.